### PR TITLE
Removed mobileWidth from NavLinks

### PR DESCRIPTION
**Pull request by 3t8**
Removed mobileWidth parameter #51

### CODE REVIEW

File: front/src/components/header/Header.tsx [hunk 1]
```diff
@@ -93,7 +91,7 @@ const Header = ({ isLoggedIn, setIsLoggedOut }: Props) => {
         mr={1}
         mt={{ base: 4, md: 0 }}
         padding={{ base: 0, md: '0 12px 5px' }}
-        width={{ base: mobileWidth, md: 'unset' }}
+        width={{ base: 0, md: 'unset' }}
```
Comment (trybick) [inline front/src/components/header/Header.tsx]: I think we can remove this whole line, so we don't even have width here at all, if you don't mind.

Comment (3t8) [reply front/src/components/header/Header.tsx]: Yes, that's even better

Comment (trybick) [reply front/src/components/header/Header.tsx]: Also some changes went in that changed 'padding' to 'p' so if you see a merge conflict that's why.

Comment (trybick) [reply front/src/components/header/Header.tsx]: Not trying to rush :) but you will remove the widths lines right?

**Pull request merged**